src/sketchup-stl.rb

```diff
@@ -52,7 +52,7 @@ def self.translate(string)
       'This is an open source project sponsored by the SketchUp team. More ' <<
       'info and updates at https://github.com/SketchUp/sketchup-stl'
     )
-    extension.version = '2.1.6'
```

Comment:
squash-merge  - is that why I am not seeing my branches merge with Sketchup branches after I fetch?